package.json

```diff
@@ -257,16 +254,21 @@
     "source-map-support": "^0.5.6",
     "ts-node": "^7.0.0",
     "tslint": "^5.10.0",
-    "typescript": "^2.9.1",
+    "typescript": "^3.0.0",
+    "vscode": "^1.1.32",
```

Comment:
`vscode` needs to be removed since we moved to `vscode-test` + `@types/vscode` - causes build errors otherwise.

See: https://code.visualstudio.com/api/working-with-extensions/testing-extension#migrating-from-vscode

Reply:
good catch I think rebasing on master should fix this (as it is correct on master)

Reply:
Thanks, I've merged in master now.